src/containers/shared/components/NFTokenLink.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+
+export interface NFTokenLinkProps {
+  link?: boolean
```

Comment:
Where is this set to true?

Reply:
it looks like it's defaulted to true regardless. so where is it set to _false_ is the question? And why would be it be set to false?

also, this isn't a good name for an argument. `showLink` or `displayLink` would be better. Furthermore, the name of the component - `NFTokenLink` tells me that this is always a link. Why would I use this without a link?

Reply:
yeah fair, i was thinking of following the same convention used in `Account` component. but i dont think non-link will be used anywhere. I will remove it